data/remnant/remnant.txt

```diff
@@ -276,6 +276,498 @@ phrase "remnant"
+		"They have a longer range than your outfit scanner"
+		"Those babies scan fast"
+		"They are a must for your plundering needs"
+	word
+		"."
+		"!"
+
+phrase "friendly remnant"
+	word
+		"Have you heard"
+		"Do you know"
+		"Are you familiar"
+	word
+		" "
+	word
+		"about"
+		"regarding"
+	word
+		" "
+	word
+		"the"
+		"those"
+	word
+		" "
+	word
+		"Quarg"
+		"Pug"
+		"Hai"
+		"Drak"
+		"Archons"
+		"insects far west of our space"
+	word
+		"?"
+	word
+		" "
+	phrase
+		"friendly remnant (aliens.!)"
+		"friendly remnant (aliens?)"
+
+phrase "friendly remnant (aliens.!)"
+	word
+		"They look strong"
+		"They look weak"
+		"Maybe we should contact them"
+		"I think they are dangerous"
+		"I think they are good"
+		"I think we can trust them"
+		"I want to plunder their tech"
+		"I want their ships"
+		"I have seen them"
+		"I haven't seen them yet"
+		"I wonder if the prefects already knew about them"
+	word
+		"."
+		"!"
+
+phrase "friendly remnant (aliens?)"
+	word
+		"Are they strong"
+		"Are they weak"
+		"Are they friendly"
+		"Are they hostile"
+		"How strong are they"
+		"What are the odds of capturing their ships"
+		"Did your government met them"
+		"Is it worth plundering them"
+	word
+		"?"
+
+phrase "friendly remnant"
+	word
+		"Be careful"
+		"Stay alert"
+		"Be vigilant"
+	word
+		" "
+	word
+		"about"
+		"aboot"
+	word
+		" "
+	word
+		"those"
+		"the"
+	word
+		"Alphas in this galaxy"
+		"pirates in your space"
+		"Korath raiders"
+		"Korath"
+		"Archon in Nenia
+	word
+		"."
+		"!"
+	word
+		" "
+	word
+		"You don't want to kill yourself"
+		"It is dangerous to encounter them"
+		"Always have the best possible weapons with you"
+		"We have ships that can cloak just in case"
+	word
+		"."
+		"!"
+
+phrase "hostile remnant"
+	word
+		"(It only consists of signs so deep that a regular human can't even understand. Seems like they don't like you.)"
+
+phrase "hostile remnant"
+	word
+		"Apparently"
+		"It seems"
+		"I think"
+	word
+		" "
+	word
+		"you are"
+	word
+		"not different to"
+		"very similar to"
+		"like"
+	word
+		"them"
+		"an Alpha"
+		"a genetically modified weapon"
+		"a Korath"
+	word
+		"."
+	word
+		" "
+	word
+		"Too bad"
+		"Too sad"
+		"I hope you think about your actions"
+		"You backstabbed us"
+		"You lied to us"
+		"You violated our trust"
+	word
+		"."
+
+phrase "hostile remnant (we're)"
+	word
+		"we're"
+	word
+		" "
+	word
+		"friends"
+		"comrades"
+		"brethens"
+		"good"
+		"awesome"
+		"cool"
+
+phrase "hostile remnant (you're)"
+	word
+		"you're"
+	word
+		" "
+	word
+		"not an Alpha"
+		"not hostile
+		"friendly"
+		"good"
+		"human"
+		
+
+phrase "hostile remnant"
+	word
+		"I"
+		"we"
+	word
+		" "
+	word
+		"thought"
+	word
+		" "
+	phrase
+		"hostile remnant (we're)"
+		"hostile remnant (you're)
+	word
+		"."
+	word
+		" "
+	word
+		"Why do you have to do this"
+		"Are you insane"
+		"Did I hurt you"
+		"Perhaps we intimidated you"
```

Comment:
1. `Perhaps we intimidated you` is asked in a questioning way, but isn't a question, it's a statement.
2. If you were intimidated by someone, you probably wouldn't attack them, so I don't know if this line makes sense.